requirements.txt

```diff
@@ -4,5 +4,6 @@ numpy
 phasespace
 progress
 pyyaml
+sympy
 tensorflow==2.1.*
 xmltodict
```

Comment:
`xmltodict` can indeed be removed (also from `setup.py`!)

Reply:
1bf2ee2ba9cb0527d925a6388fa2b82a835e51e2